### DISCUSSION TITLE
a few questions from a new user

### DISCUSSION THREAD

**Discussion by kira1000** [karashiiro/TextToTalk#82] [Q&A]
Hello, I just found out about this brilliant plugin yesterday and I have a couple of questions.
1. Is there a way to get texttotalk to read NPC chatter (not the quest dialogue, the random things NPCs say)?  I noticed the game wasn't doing so when I entered the Archer's guild and the archers were talking but the plugin didn't trigger.
2. Is it possible (using exceptions) to get the mod to ignore the enter/exit sanctuary messages?  Want to hear the quest progression but not the constant sanctuary messages in towns etc.
Thanks in advance, keep up the amazing work.

**Comment by johnysandels** [comment]
Hi there! 
For 1st, I'd suggest Turing on NPC Announcements in the channels sections. That might be what you're talking about.
For the 2nd, there is a triggers/exclusions section, where it will exclude any sentence with the text in it.

**Reply by kira1000** [reply]
> Hi there! For 1st, I'd suggest Turing on NPC Announcements in the channels sections. That might be what you're talking about. For the 2nd, there is a triggers/exclusions section, where it will exclude any sentence with the text in it.
That did not work sadly, already had NPC announcements open but nothing happened.  Could it be because the NPC is nameless or the fact I am using the speech bubble mod?

**Reply by kira1000** [reply]
I THINK I found out why-apparently the NPC chatter doesn't appear in chat, which would explain why the mod doesn't pick it up.

**Reply by johnysandels** [reply]
Yeah, I was pretty sure that chatter doesn't get sent to your chat, so it couldn't be read. It was worth a try though!

**Reply by karashiiro** [reply]
If you mean the text in NPC chat bubbles, that usually isn’t logged, so the plugin can’t read it.

**Reply by kira-sakaki** [reply]
https://github.com/PunishedPineapple/WhatDidYouSay
Dalamud/XIVLauncher plugin to print NPC speech bubbles to the chat log.

**Reply by kira-sakaki** [reply]
However, there may be duplicates.
(Example:Aloalo Island npc channel)
